src/script/lua_api/l_camera.cpp

```diff
@@ -115,38 +120,40 @@ int LuaCamera::l_get_offset(lua_State *L)
 	return 1;
 }
 
+// get_look_dir(self)
```

Comment:
`// FIXME: wouldn't localplayer be a better place for this?`

for the sake of completeness